src/main/java/duke/logic/commands/DeleteCommand.java

```diff
@@ -30,9 +30,9 @@ public DeleteCommand(int index) {
     @Override
     public CommandResultText execute(Model model) throws DukeException {
         try {
-            Task task = model.getTasks().remove(index);
+            Event event = model.getEvents().remove(index);
             model.save();
-            return new CommandResultText(MESSAGE_DELETE + task);
+            return new CommandResultText(MESSAGE_DELETE + event);
         } catch (IndexOutOfBoundsException e) {
             throw new DukeException(Messages.ERROR_INDEX_OUT_OF_BOUNDS);
```

Comment:
Perhaps we need a specific exception for index out of bounds.
We should be also able to know what was the query that caused the exception too.

Reply:
👌